Library/PTPusher.m

```diff
@@ -274,6 +274,20 @@ - (void)subscribeAll
   }
 }
 
+- (void)unsubscribeAllChannels
+{
+  if (channels && channels.count) {
+	NSMutableArray *channelsToUnsubscribe = [[NSMutableArray alloc] init];
+	  for (PTPusherChannel *channel in [channels allValues]) {
+		[channelsToUnsubscribe addObject:channel];
```

Comment:
Do we need the intermediate `channelsToUnsubscribe` array?

Can we not just put `[channel unsubscribe];` here?